kwok/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.k8s.io/kwok/cluster:v0.3.0-k8s.v1.27.3
+
+COPY --chmod=755 user-signups.sh entrypoint.sh toolchain.dev.openshift.com_usersignups.yaml ./
+
+RUN KWOK_KUBE_APISERVER_PORT=0 kwokctl create cluster "$@" || exit 1 && \
+    kwokctl kubectl apply -f toolchain.dev.openshift.com_usersignups.yaml && \
+    sh ./user-signups.sh && \
```

Comment:
instead of uploading a script like this and running it doing the build, we can just the script once and commit the results to the repo as static YAML files. That way we don't have an unnecessary script left behind in the image and we can more easily customize the records later.